Encodings.md

```diff
@@ -179,6 +179,12 @@ This encoding is adapted from the Binary packing described in
 ["Decoding billions of integers per second through vectorization"](http://arxiv.org/pdf/1209.2137v5.pdf)
 by D. Lemire and L. Boytsov.
 
+Delta encoding is best when used on sorted data, or data with runs of repeated
+values. It can also be useful when the range of values is small, such as would
+be the case with INT_8 data. It should *not* be used when the range of the data
+would necessitate the use of large bitwidths, as could be the case with random
+INT32 values.
```

Comment:
Just curious that int32 and int8 spelling is different.

Reply:
I took `INT_8` from the logical type name, and `INT32` from the physical type name.  That distinction could be made clearer.